### PR TITLE
Updating the example of SELECT

### LINKED ISSUE #221: Update SELECT: support multiple conditions in where clause
### Related code PR

https://github.com/singularity-data/risingwave/pull/3756

### Which part(s) of the docs might be affected or should be updated? And how?

Can update the examples in https://www.risingwave.dev/docs/latest/sql-select/.
Skip Release Notes.

### Reference

_No response_

### PR DESCRIPTION
**Pull request by CharlieSYH**
## Info
 Updating the example to reflect the support for multiple condions in the WHERE clause.

- **Related code PR**: 
https://github.com/risingwavelabs/risingwave/pull/3756

- **Related doc issue**: 
Resolves https://github.com/risingwavelabs/risingwave-docs/issues/221

## Before merging
  - [ ] (For version-specific PR) I have selected the corresponding software version in **Milestone** and linked the related doc issue to this PR in **Development**.
  - [x] I have acquired the approval from the owner (and optionally the reviewers) of the code PR and at least one tech writer (`bernscode`, `CharlieSYH`, `emile-00`, & `hengm3467`). 
  - [ ] I have checked the doc site preview and the updated parts look all good. <details><summary>How?</summary>image</details>
  - [ ] (For version-specific PR) I have confirmed that this PR is for a released version. If the software version is not released, put it on hold.

## Published doc pages
  [ After merging this PR, edit the description to include the links to the updated doc pages here. For example, https://www.risingwave.dev/docs/latest/intro/. ]

### CODE REVIEW

**Review by xxchan — COMMENTED** [review]
> Updating the example to reflect the support for multiple condions in the WHERE clause.

This is already supported previously. https://github.com/risingwavelabs/risingwave/pull/3756 only makes ~~it~~ slightly more efficient 👀

To be more specifically, "scan range" is used to push some condition in the WHERE clause into the scan operator. It's a performance optimization, so not a (direct) "user-facing-change"? 

And this PR makes the condition `in (1,2,3)` becomes more efficient, by transforming it into multiple "scan ranges".

File: docs/sql/commands/sql-select.md [hunk 1]
```diff
@@ -74,18 +75,19 @@ The table `company` includes the columns `company_id` and `taxi_id`, where `taxi
...
+    sum(trips.distance) AS total_distance, 
+    sum(trips.duration) AS total_duration
+FROM taxi_trips AS trips
 LEFT JOIN taxi ON trips.id = taxi.trip_id
 WHERE taxi_id IN
     (SELECT taxi_id
-    FROM company_id
+    FROM company
     WHERE company_id = 'Yellow Taxi')
+    AND fare > 2.50
```
Comment (xxchan) [inline docs/sql/commands/sql-select.md]: a `)` is missing?

**Comment by xxchan** [comment]
"Scan range" is used to push some condition in the WHERE clause into the scan operator. It's a performance optimization, so not a (direct) "user-facing-change"?

**Comment by CharlieSYH** [comment]
> > Updating the example to reflect the support for multiple condions in the WHERE clause.
> 
> This is already supported previously. [risingwavelabs/risingwave#3756](https://github.com/risingwavelabs/risingwave/pull/3756) only makes ~it~ slightly more efficient 👀
> 
> To be more specifically, "scan range" is used to push some condition in the WHERE clause into the scan operator. It's a performance optimization, so not a (direct) "user-facing-change"?
> 
> And this PR makes the condition `in (1,2,3)` becomes more efficient, by transforming it into multiple "scan ranges".

Thanks for the clarification! It really helps me understand the code changes better. Since we did not mention this in the release notes, users won't take it as a new feature.

File: docs/sql/commands/sql-select.md [hunk 2]
```diff
@@ -49,43 +49,45 @@ Where `from_item` can be:
...
 SELECT 
     taxi.taxi_id, 
-    sum(trips.distance) as total_distance, 
-    sum(trips.duration) as total_duration
-FROM taxi_trips as trips
+    sum(trips.distance) AS total_distance, 
+    sum(trips.duration) AS total_duration
+FROM taxi_trips AS trips
 LEFT JOIN taxi ON trips.id = taxi.trip_id
 WHERE taxi_id IN
```
Comment (xxchan) [inline docs/sql/commands/sql-select.md]: Would this look slightly better?
```sql
WHERE taxi_id IN (SELECT taxi_id
                  FROM company
                  WHERE company_id IN ('Yellow Taxi', 'FabCab'))
      AND trips.fare > 2.50
```
or something like
```sql
WHERE taxi_id IN (
          SELECT taxi_id
          FROM company
          WHERE company_id IN ('Yellow Taxi', 'FabCab')
      )
      AND trips.fare > 2.50
```

**Review by xxchan — APPROVED** [review]
Others LGTM

File: docs/sql/commands/sql-select.md (hunk 1 above)
Comment (xxchan) [reply docs/sql/commands/sql-select.md]: Not missing. My fault

**Review by hengm3467 — APPROVED** [review]
Looks good to me.

**Pull request merged**